openassessment/assessment/api/staff.py

```diff
@@ -249,36 +248,37 @@ def get_submission_to_assess(course_id, item_id, scorer_id):
             retrieving staff workflow information.
 
     Examples:
-        >>> get_submission_to_assess("a_course_id", "an_item_id", "a_scorer_id")
-        {
+        >>> get_submission_to_assess("a_course_id", "an_item_id")
+        [{
             'student_item': 2,
             'attempt_number': 1,
             'submitted_at': datetime.datetime(2014, 1, 29, 23, 14, 52, 649284, tzinfo=<UTC>),
             'created_at': datetime.datetime(2014, 1, 29, 17, 14, 52, 668850, tzinfo=<UTC>),
             'answer': { ... }
-        }
+        }, ...]
 
     """
-    student_submission_uuid = StaffWorkflow.get_submission_for_review(course_id, item_id, scorer_id)
+
+    student_submission_uuid = StaffWorkflow.get_submission_for_review(course_id, item_id)
```

Comment:
Could you add plural part in the variable name?

Reply:
Done